frontend/kesaseteli/employer/src/components/application/steps/step3/Step3Summary.tsx

```diff
@@ -24,7 +25,26 @@ const Step3Summary: React.FC = () => {
     <ApplicationForm stepTitle={t('common:application.step3.header')}>
       <ApplicationSummary />
       <FormSection columns={1}>
-        <Checkbox id="termsAndConditions" validation={{ required: true }} />
+        <Checkbox
+          id="termsAndConditions"
+          validation={{ required: true }}
```

Comment:
Don't you need here `required` prop to add an asterisk to the label?

Reply:
hds should add with required param, yes

Reply:
It is set inside the Checkbox component:
https://github.com/City-of-Helsinki/yjdh/blob/e9015491fbb726158140af9b51d22820cefb86a8/frontend/kesaseteli/employer/src/components/application/form/Checkbox.tsx#L70